### PR TITLE
[MIG] pos_gift_ticket: Migration to 10.0

### PR DESCRIPTION
**Pull request by spacemonkeythe**

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hi @chienandalu , please we should review this PR! Thanks!

File: pos_gift_ticket/__manifest__.py
Comment (chienandalu) [inline pos_gift_ticket/__manifest__.py:23]: Please follow OCA guidelines: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: pos_gift_ticket/__manifest__.py
Comment (chienandalu) [inline pos_gift_ticket/__manifest__.py:28]: https://github.com/OCA/pos

File: pos_gift_ticket/__manifest__.py
Comment (chienandalu) [inline pos_gift_ticket/__manifest__.py:38]: Shouldn't this be `True`? :wink:

File: pos_gift_ticket/static/src/js/main.js
Comment (chienandalu) [inline pos_gift_ticket/static/src/js/main.js:13]: is this being used?

File: pos_gift_ticket/static/src/js/main.js
Comment (chienandalu) [inline pos_gift_ticket/static/src/js/main.js:43]: Better to split this in separate files wigdets.js and chrom.js, isn't it?

File: pos_gift_ticket/static/src/xml/pos.xml
Comment (chienandalu) [inline pos_gift_ticket/static/src/xml/pos.xml:30]: ?

File: pos_gift_ticket/views/pos_template.xml
Comment (chienandalu) [inline pos_gift_ticket/views/pos_template.xml:15]: missing blank line

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@spacemonkeythe Any news on this ?

**Comment by MPoch-PlanetaTIC** [comment]
I have tested this module but the button "Print Gift" just prints the original report of pos.order (with prices).
Maybe I'm missing some configuration?

File: pos_gift_ticket/static/src/js/main.js
Comment (MPoch-PlanetaTIC) [inline pos_gift_ticket/static/src/js/main.js:20]: I've edited line 20 like this:
"$('.pos-receipt-container').html(QWeb.render('PosGiftTicket',{"
to make it work properly.

File: pos_gift_ticket/static/src/js/main.js
Comment (MPoch-PlanetaTIC) [inline pos_gift_ticket/static/src/js/main.js:56]: Translation of "Print Gift" button is missing.

File: pos_gift_ticket/static/src/xml/pos.xml
Comment (MPoch-PlanetaTIC) [inline pos_gift_ticket/static/src/xml/pos.xml:26]: It would be great to put Phone, User and Shop in html tags (like div) to make it possible to inherit and personalize Gift Ticket content.

**Comment by legalsylvain** [comment]
closing, due to inactivity. Feel free to reopen.

**Pull request closed without merging**
